Makefile

```diff
@@ -54,6 +54,17 @@ push-tag: # example usage: make push-tag tag=v1.100.1 remote=origin
 	 	git push $(remote) "$${dir:2}/$(tag)"); \
 	done
 
+.PHONY: publish-godoc
+publish-godoc: # example usage: make publish-godoc tag=v1.100.1
+	@[ "$(tag)" ] || ( echo ">> 'tag' is not set"; exit 1 )
+	@echo "godoc for root"
+	@cd ; go get -u github.com/signalfx/signalfx-go-tracing@$(tag)
+	@set -e; cd; \
+	for dir in $(SUBMODULES); do \
+	  (echo godoc for "$${dir:2}" && \
```

Comment:
```suggestion
	  (echo "godoc for $${dir:2}" && \
```

Reply:
I will have to test it. I see the same issue in other Make targets.

Reply:
Done https://github.com/signalfx/signalfx-go-tracing/pull/159/commits/7c058849238db83bcc0348dc5ab6dede7e549e84